destinations/data/jaeger.yaml

```diff
@@ -20,4 +20,28 @@ spec:
       componentProps:
         type: text
         required: true
+    - name: JAEGER_CERT_PEM
+      displayName: Client Certificate
+      componentType: textarea
+      componentProps:
+        type: password
+        required: false
+        placeholder: '-----BEGIN CERTIFICATE-----'
+        tooltip: 'When using TLS, the client certificate is required.'
+    - name: JAEGER_KEY_PEM
+      displayName: Client Key
+      componentType: textarea
+      componentProps:
+        type: password
+        required: false
+        placeholder: '-----BEGIN PRIVATE KEY-----'
+        tooltip: 'When using TLS, the client key is required.'
+    - name: JAEGER_CA_PEM
+      displayName: Certificate Authority
+      componentType: textarea
+      componentProps:
+        type: password
+        required: false
+        placeholder: '-----BEGIN CERTIFICATE-----'
+        tooltip: 'When using TLS, the Certificate Authority is optional. Unless TLS is used with self-signed certificates, then the Certificate Authority is required.'
```

Comment:
Most users will not set the CA certificate in odigos, but will use some k8s tool to store and distribute it. This text makes it sound like we will not work in this case.